catalyst/utils/parser.py

```diff
@@ -74,10 +75,11 @@ def parse_args_uargs(args, unknown_args):
     args_copy = copy.deepcopy(args)
 
     # load params
-    config = {}
+    configs = []
     for config_path in args_copy.configs:
-        config_part = load_config(config_path, ordered=True)
-        config = merge_dicts(config, config_part)
+        configs.append(OmegaConf.load(config_path))
+    cli_config = OmegaConf.from_cli()
+    config = OmegaConf.merge(*configs, cli_config)
```

Comment:
could you please provide an example of usage for the new functionality?

Reply:
`catalyst-dl run --configs=config1.yaml,config2.yaml args.expdir=/path/to/src --logdir=/path/to/logs stages.data_params.batch_size=32 stages.train.state_params.num_epochs=40 --stages/train/callback_params/saver/save_n_best=2:int`
where can I write this?

Reply:
I think you could add this to our pipelines test cases...
we have a bunch of them 😄 
https://github.com/catalyst-team/catalyst/blob/master/bin/tests/check_dl_cv.sh